packages/@okta/vuepress-theme-prose/global-components/SignUpOiePreview.vue

```diff
@@ -0,0 +1,429 @@
+<template>
```

Comment:
I decided to duplicate and modify the `SignUp.vue` component since trying to shoehorn the behavior needed for this work was getting messy, plus this preview signup form should only be needed for a brief period of time so removing it will be a lot easier if it's separate.